### PR TITLE
AB test MPR/ASR dynamic-agg, fix  dd issues

### PR DESCRIPTION
**Pull request by sravfeyn**
@emord

### CODE REVIEW

File: custom/icds_reports/utils/__init__.py [hunk 1]
```diff
@@ -231,12 +231,13 @@ def _custom_data(self, selected_location, domain):
                 loc_type = selected_location.location_type.name
             else:
                 loc_type = None
+            tags = ["location_type:{}".format(loc_type), "report_slug:{}".format(self.slug)]
+            if allow_conditional_agg:
+                tags.append("allow_conditional_agg:yes")
             datadog_histogram(
                 "commcare.icds.block_reports.ucr_querytime",
                 timer.duration,
-                tags="config:{}, location_type:{}, report_slug:{}".format(
```
Comment (emord) [inline custom/icds_reports/utils/__init__.py]: was it intentional to leave config out of this?

**Review by emord — APPROVED** [review]

File: custom/icds_reports/utils/__init__.py (hunk 1 above)
Comment (sravfeyn) [reply custom/icds_reports/utils/__init__.py]: Thanks for catching, was an oversight!

File: custom/icds_reports/utils/__init__.py [hunk 2]
```diff
@@ -231,12 +231,13 @@ def _custom_data(self, selected_location, domain):
                 loc_type = selected_location.location_type.name
             else:
                 loc_type = None
+            tags = ["location_type:{}".format(loc_type), "report_slug:{}".format(self.slug), "config:{}".format(config['id'])]
```
[the remark this answers is not in the document]
Comment (emord) [reply custom/icds_reports/utils/__init__.py]: could set these variables on separate lines to avoid this and make it more readable

**Review by emord — APPROVED** [review]

File: custom/icds_reports/utils/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sravfeyn) [reply custom/icds_reports/utils/__init__.py]: Considered that, but some vars need to be passed around.

**Pull request merged**